app/setup.py

```diff
@@ -90,22 +90,25 @@ def create_app(  # noqa: C901  pylint: disable=too-complex, too-many-statements
 ):
     application = Flask(__name__, template_folder="../templates")
     application.config.from_object(settings)
+    if setting_overrides:
+        application.config.update(setting_overrides)
     application.eq = {}
 
     with open(application.config["EQ_SECRETS_FILE"]) as secrets_file:
         secrets = yaml.safe_load(secrets_file)
+    expected_secrets = (
+        ["ADDRESS_LOOKUP_API_AUTH_TOKEN_SECRET"]
+        if application.config["ADDRESS_LOOKUP_API_AUTH_ENABLED"]
+        else []
+    )
```

Comment:
I saw this as something that would get added to with any future conditional required secrets.

Reply:
Would it be better if the variable was named `conditional_required_secrets`?

Reply:
> I saw this as something that would get added to with any future conditional required secrets.

Fair enough. It was cause the of the ternary operator than an if block as I expected it to be like:
```py
expected_secrets = []
if application.config["ADDRESS_LOOKUP_API_AUTH_ENABLED"]:
	expected_secrets.append("ADDRESS_LOOKUP_API_AUTH_TOKEN_SECRET")
```
But since there is only one, happy to keep as ternary.

> Would it be better if the variable was named conditional_required_secrets?

I would be happy with that.

Reply:
Actually, seeing the `if` block I think that's clearer than the ternary. I'll change it to that 😄 .